SpatialGDK/Source/SpatialGDK/Private/Interop/SpatialSender.cpp

```diff
@@ -96,6 +96,10 @@ Worker_RequestId USpatialSender::CreateEntity(USpatialActorChannel* Channel, uin
 
 	ComponentDatas.Add(ComponentPresence(EntityFactory::GetComponentPresenceList(ComponentDatas)).CreateComponentPresenceData());
 
+	// Add Actor completeness tags.
+	// This has to come after the component presence list, so that Actor completeness tags don't get delegated.
+	ComponentDatas.Add(ComponentFactory::CreateEmptyComponentData(SpatialConstants::LB_TAG_COMPONENT_ID));
```

Comment:
Alternatively we could add filtering to CreateComponentPresenceData that makes sure it only includes generated components etc. This all removed with component sets so might be moot though.

Reply:
we could also make the LBEnforcer skip these tags when updating component presence data. IMO it would be nicer if it was done that way round as it makes no assumptions about how the component presence component is constructed. Right now we assume that component presence is not exactly the set of components present on the entity, but some somewhat arbitrary "useful" set.

Same point on component sets though so I think it is moot